projects/plugins/jetpack/modules/search/instant-search/components/search-results.jsx

```diff
@@ -90,7 +89,7 @@ class SearchResults extends Component {
 	renderPrimarySection() {
 		const { highlightColor, searchQuery } = this.props;
 		const { results = [], total = 0, corrected_query = false } = this.props.response;
-		const textColor = useMemo( () => getConstrastingColor( highlightColor ), [ highlightColor ] );
```

Comment:
Why does this PR remove useMemo from `preact/hooks` here? 😕

Reply:
Ah, it's because it can't be invoked within a class component 👍

Reply:
Yeah. I found some mentions online about using `getDerivedStateFromProps` to do something similar for a class component, but then I figured if we did need caching of the computed value then someone would say so in review.